pkgs/by-name/ho/holesail/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+buildNpmPackage rec {
+  pname = "holesail";
+  version = "1.10.1";
+
+  src = fetchFromGitHub {
+    owner = "holesail";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-eANDS4Avu6or+1XXRmM+rceZzjKchhoc5nbv1grpvwE=";
+  };
+
+  npmDepsHash = "sha256-TKxm8WdsJ6fbs8WBAN7yxNQoJ43fTQghD0KYGxRRd50=";
+
+  dontNpmBuild = true;
+
+  meta = with lib; {
+    description = "Peer-to-peer network tunneling and reverse proxy software.";
```

```suggestion
    description = "Peer-to-peer network tunneling and reverse proxy software";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes